### ISSUE TITLE
Problem/Bug Harvest Console

### ISSUE THREAD

**Issue by manilly** [INSPIRE-MIF/helpdesk-geoportal#33]
Hi,
I started a new harvest, and there is again a problem with the harvest console. There are no harvest numbers nor is there a report for the metadata with linkage issues or indicators.

![image](https://user-images.githubusercontent.com/45065613/113874910-8cfa7b80-97b6-11eb-85bf-8f6c3675d0f6.png)

![image](https://user-images.githubusercontent.com/45065613/113874984-9c79c480-97b6-11eb-9c06-d88ee7a76724.png)

![image](https://user-images.githubusercontent.com/45065613/113875080-b0bdc180-97b6-11eb-9b12-2789eade6f0e.png)

**Comment by dartasensi** [comment]
Dear @manilly 

thanks for the feedback. I'm on it right now.
I checked and I see your harvest data. Also SI harvest manifest a similar issue, while ES encounters some issues with completing the harvest. In your case, the issue is centred on our Solr instance, that it seems to prevent returning results, as you clearly noticed.
I keep you posted.

KR
Davide

**Comment by dartasensi** [comment]
Dear @manilly 

I finally found that, in our sandbox environment, there was a significant corruption of data regarding several harvest sessions (executed between April 1st and today) that led to various errors reported here in the Helpdesk.

Therefore, the only solution was to stop the services, then carefully execute a clean up from all of these errors before proceeding further.
Even if the successful AT harvested data (ran on April 7th ~h14:37) is still stored on our disks, I preferred to start again from scratch so I scheduled a new harvest for your endpoint, that it should complete tomorrow and finally restore the situation.

Thanks for your patience.

KR
Davide
